pages/_document.js

```diff
@@ -44,7 +44,7 @@ export default class extends Document {
           { this.helmetHeadComponents }
         </Head>
         <body {...this.helmetBodyAttrComponents}>
-          <style jsx global>{styles}</style>
+          <style>{styles}</style>
```

Comment:
it'll lose some benefits of `styled-jsx` (like prefixing). I think there's something wrong in `getInitialProps`?
which does not inject styles properly
https://github.com/zeit/next.js/issues/2593

Reply:
I think https://github.com/zeit/next.js/issues/2593 is a different issue, which is unable to get style. However, in our case, we can properly get style but failed to inject to the document

Reply:
This issue https://github.com/zeit/next.js/issues/885 pretty much explains why we should use `<style>` instead of styled-jsx in custom document